src/locale/misc/views/en.yml

```diff
@@ -37,6 +37,18 @@ columnDialog:
       description: Add your choice of fields like email, phone number, etc.
       keywords: field multiple
       title: Person fields
+    surveyResponse:
+      allOptions: All Options (Single Column)
+      allOptionsHeader: All Options
+      allOptionsSeparated: All Options (Separate Columns)
+      description: Include response to a question from any survey.
+      keywords: field multiple
```

Comment:
This will need some more work, but that's what we'll be doing in the separate issue that @ziggabyte is creating, so no need to do anything about it here.